src/Library/WizardControl.cs

```diff
@@ -400,12 +403,17 @@ private void RemoveSystemEvents()
 		protected override void OnParentChanged(EventArgs e)
 		{
 			base.OnParentChanged(e);
-			if (parentForm != null)
-				parentForm.Load -= parentForm_Load;
-			parentForm = Parent as Form; // FindForm();
+			if (parentAsForm!=null)
```

Comment:
`if (parentControl is Form parentAsForm)`

Reply:
Fixed, I accidentally created a new pull request. This one can be rejected.